Automattic/wp-calypso
# Reader: remove site blocks Flux store

Now that site blocks have been ported to Redux (https://github.com/Automattic/wp-calypso/pull/10944), we can safely remove the old Flux store, `lib/reader-site-blocks`.

This PR also removes `reader/post-errors`, an unused component that included the old Flux store.

### To test

Ensure that the Reader functions normally.

Ensure that site blocking works in the Reader (unless you're @samouri, for whom it is broken always).
